### PR TITLE
Added keyfile option to docklet_shinyapp function a.o.

### PR DESCRIPTION
**Pull request by pjastam**
The docklet_shinyapp function did not have a keyfile argument and assumed the default private keyname id_rsa. I added a keyfile argument to this function (and related functions) in order to be able to set non-default private keynames. I tested the new code locally and it worked for me. This may be interesting for others too :-)

### CODE REVIEW

**Comment by pachadotdev** [comment]
> The docklet_shinyapp function did not have a keyfile argument and assumed the default private keyname id_rsa. I added a keyfile argument to this function (and related functions) in order to be able to set non-default private keynames. I tested the new code locally and it worked for me. This may be interesting for others too :-)

thanks a lot, and sorry for the slow reply
I'll test it at some point during the weekend

File: R/docklet.R [hunk 1]
```diff
@@ -56,11 +56,11 @@
 #' instance, you can construct like \code{http://<ip address>:<port>} where
 #' IP address can most likely be found like \code{d$networks$v4[[1]]$ip_address}
 #' and the port is the port you set in the function call.
-#' 
+#'
```
Comment (pachadotdev) [inline R/docklet.R:59]:
```suggestion
#' 
```

**Review by pjastam — COMMENTED** [review]
Looks fine to me!

**Comment by pachadotdev** [comment]
> Looks fine to me!

@pjastam hi, can you please commit the suggestions I sent

**Review by pjastam — COMMENTED** [review]
@pachadotdev hi, I committed your suggestions to the test-shinyapp branch in my fork of the original analogue repository. It looks like I cannot merge the 2 commits into [pachadotdev:main](https://github.com/pachadotdev/analogsea) from [pjastam:test-shinyapp](https://github.com/pjastam/analogsea/tree/test-shinyapp), because I do not have write access I suppose? Any help would be greatly appreciated :-)

**Comment by pachadotdev** [comment]
> @pachadotdev hi, I committed your suggestions to the test-shinyapp branch in my fork of the original analogue repository. It looks like I cannot merge the 2 commits into [pachadotdev:main](https://github.com/pachadotdev/analogsea) from [pjastam:test-shinyapp](https://github.com/pjastam/analogsea/tree/test-shinyapp), because I do not have write access I suppose? Any help would be greatly appreciated :-)

thanks!
there's a problem with the CI that I won't be able to fix until 10-15 days from now, when it's fixed I'll merge

**Comment by pachadotdev** [comment]
I had to close the PR because my suggestions were not merged. I ended up implemented the idea that you described.
As constructive feedback, this is a good idea but the indentation changes made it really hard to see the actual changes.

**Pull request closed without merging**